docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -33,7 +35,7 @@ Product p = JsonSerializer.Deserialize<Product>(@"{""Name"":""Banana"",""ExpiryD
 Console.WriteLine(p.Name); // Banana
 Console.WriteLine(p.ExpiryDate); // 7/26/2019 12:00:00 AM
 
-// With default options, input DateTimes must conform to the extended ISO 8601-1:2019 profile.
+// With default options, input DateTimes must conform to the native implementation of the ISO 8601-1:2019 profile.
```

Comment:
Why was the "extended" qualifier removed?

Reply:
reverted